packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -6,6 +6,11 @@ import { ISemanticTextColors } from './ISemanticTextColors';
  * The collection of all semantic slots for colors used in themes.
  *
  * Note: text colors are defined in ISemanticTextColors.ts.
+ * The plan is to move completely to semantic slots and replace all usage of fabric palette slots.
+ * We still have a ways to go to complete this plan and we've been actively making some of these changes.
+ * At present, we're not adding any more slots to avoid bloating the already sizeable offering.
+ * We're also working on a better solution for design tokens overall in our next iteration, Fluent.
```

Comment:
Not sure we should publicly talk about Fluent yet. (Not to mention, it's an overloaded term right now!)